src/YarpUtilities/include/WalkingControllers/YarpUtilities/TransformHelper.h

```diff
@@ -0,0 +1,86 @@
+// SPDX-FileCopyrightText: Fondazione Istituto Italiano di Tecnologia (IIT)
```

Comment:
Do you think it is worth moving this class to blf?

Reply:
To be honest, I think it is not strictly necessary. In the end, I open two yarp devices. The ``iFrameTransform `` interface is quite clean and after https://github.com/robotology/yarp/pull/3162 there are also the bindings. I moved it to a file to avoid cluttering the ``Module.h`` file. In any case, if you think of a specific usage in blf, I am open to that too 😉